app/pages/project/instances/instance/tabs/StorageTab.tsx

```diff
@@ -120,29 +135,98 @@ export function StorageTab() {
 
   const { data: instance } = usePrefetchedApiQuery('instanceView', instancePathQuery)
 
-  const makeActions = useCallback(
+  const { mutateAsync: instanceUpdate } = useApiMutation('instanceUpdate', {
+    onSuccess() {
+      apiQueryClient.invalidateQueries('instanceView')
```

Comment:
perhaps add an `addToast({ content: 'Your instance has been updated' })` here?

Reply:
Good idea, though I think it would be even better if it was specific to the action — this is used for both set and unset boot disk. So it needs to be triggered at each call site. That might require moving this invalidate there because I think if you pass `onSuccess` as an option at call time it will override this one.